packages/sui-changelog/README.md

```diff
@@ -0,0 +1,63 @@
+# sui-changelog
+
+> CLI to retrieve a changelog from a set of dependencies.
+
+## Description
+
+The main aim of this package is to build a changelog file with all the last changes you get in your last installation within your project. Dependencies are retrieved by its scope in the `node_modules` folder, so you can customize the scopes to get the changelog data from.
+
+## Usage
+
+Install `node@11` with `npm@6` in your env and run the following CLI command:
+
+```sh
+$ npx @s-ui/changelog
+```
+
+Or, if you already have it installed:
+
+```sh
+$ sui-changelog
+```
+
+### Adding more scopes
+
+By default, `sui-changelog` works only with `@s-ui` scoped packages, so if you want to add more scopes to your changelog, just add them in your project `package.json` file, the same way as the example below:
+
+```js
+{
+  "sui-changelog": {
```

Comment:
In order to follow the @s-ui convention, can we use the `config` key and, inside, use the `sui-changelog` object?

Reply:
That's a good point, but I followed the `sui-bundler` example, maybe we should create an issue for moving this change to every `sui` package.

Reply:
You're right. @s-ui/bundler is using directly the root 🤔. Could we use the `config` for this package and we will move later the @s-ui/studio? 👏

Reply:
Yep, I'm doing it right for this package 😉

Reply:
Thanks kiko! 🙇‍♂️